### DISCUSSION TITLE
Inquiry about Providing RISC-V Provider in ONNX Runtime

### DISCUSSION THREAD

**Discussion by ryansoq** [microsoft/onnxruntime#19019] [Other Q&A] [answered]
Hi all,

I have recently completed porting a RISC-V provider and would like to share this achievement with the community to potentially integrate it into the ONNX Runtime. Before proceeding, I would like to seek your opinions.

1. Is it possible for me to offer the RISC-V provider in ONNX Runtime?
2. If yes, how should I go about this process?
3. What are the relevant considerations or points to be aware of?

Thank you all for your assistance and insights!

**Comment by faxu** [comment, marked as answer]
You are welcome to contribute as a PR. There is already this PR in progress: https://github.com/microsoft/onnxruntime/pull/18115
